src/acl/FilledChecklist.cc

```diff
@@ -76,7 +76,7 @@ showDebugWarning(const char *msg)
         return;
 
     ++count;
-    debugs(28, Important(58), "ALE missing " << msg);
+    debugs(28, Important(58), "ERROR: ALE missing " << msg);
```

Comment:
This upgrade is a problem. This function is clearly intended to show *warnings*.

Reply:
There is a problem in this code indeed, but not in the added `ERROR` label. The problem is a bad function name! The reported problem is an error, and often pretty important one. The missing ALE field likely leads to incorrect Squid decisions (e.g., an `http_access` mismatch).